editor/src/components/canvas/controls/select-mode/scene-label.tsx

```diff
@@ -229,8 +227,15 @@ const SceneLabel = React.memo<SceneLabelProps>((props) => {
             backgroundColor: backgroundColor,
           }}
         >
-          {label}
-        </div>
+          <div
+            style={{
+              padding: `${4 / scale}px`,
+              fontWeight: 'bold',
```

Comment:
could you use numbers here? it makes searching for those across the repo a lot easier. Also, are you sure this should be bold - I don't think we use 700 anywhere, typically 600 is our max